app/src/main/index.ts

```diff
@@ -44,6 +52,10 @@ app.whenReady().then(() => {
     optimizer.watchWindowShortcuts(window);
   });
 
+  ipcMain.handle("getVersion", async (_event) => {
```

Comment:
since the version is mostly a constant, I wonder if we could instead just set the version as an environment variable and read that? something like adding:
```
"VITE_APP_VERSION=$npm_package_version"
"VITE_APP_NAME=$npm_package_name"
```

to the env vars in `run.sh`, and then just referencing `import.meta.env.VITE_APP_VERSION`

Reply:
I figured out how to get this working! We can't just add it to `run.sh` because that's only executed in dev mode. We also need to set this env var when running the `pnpm run build` commands. So encapsulated some code in a script that pulls the version from package.json, so we still only need to set the version string in one place.